### PR TITLE
[DOCS] Add more run options to README

### PR DESCRIPTION
**Pull request by symdec**
Other running elasticsearch possibilities added in REAME.asciidoc to help new users to run elasticsearch as daemon, service etc.

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -37,6 +37,18 @@ First of all, DON'T PANIC. It will take 5 minutes to get the gist of what Elasti
...
+
+* Running Elasticsearch as a daemon and get its process ID in the file 'pidfile' : `bin/elasticsearch -d -p pidfile`
+
+* Running Elasticsearch with SysV init : `sudo -i service elasticsearch start`
+* Can be stopped using `sudo -i service elasticsearch stop`
+
+* Running Elasticsearch with systemd : `sudo systemctl start elasticsearch.service`
+* Can be stopped using `sudo systemctl stop elasticsearch.service`
+
+* You can configure SysV / systemd to run elasticsearch when your system boots up, see https://www.elastic.co/guide/en/elasticsearch/reference/current/starting-elasticsearch.html for more details.
```
Comment (jrodewig) [inline README.asciidoc:50]:
```suggestion
For more options, see
https://www.elastic.co/guide/en/elasticsearch/reference/current/starting-elasticsearch.html[Starting
Elasticsearch].
```

**Review by jrodewig — CHANGES_REQUESTED** [review]
Thanks for this PR @KyN76.

The README is intended to be more of a quick start and not exhaustive. Most of this info is already covered in https://www.elastic.co/guide/en/elasticsearch/reference/current/starting-elasticsearch.html. I worry that including it here could eventually lead to drift and errors.

Rather than duplicate more content, I think we can link users to those docs. I've added a related suggestion.

Let me know what you think. If you agree with the suggestion, I can help you get this PR merged. Thanks again!

File: README.asciidoc (hunk 1 above)
Comment (KyN76) [reply README.asciidoc:50]: Thanks for your answer ! Your suggestion is indeed more relevant to keep the quick start as clear as possible. Then, I can only approve that !

**Comment by KyN76** [comment]
Thanks for your suggestion !

**Review by jrodewig — APPROVED** [review]
LGTM. Thanks again @KyN76.

**Comment by jrodewig** [comment]
@elasticmachine ok to test

**Comment by jrodewig** [comment]
@elasticmachine update branch

**Pull request merged**